Makefile

```diff
@@ -71,7 +72,7 @@ build-full:
 
 	cd nxcompshad && autoconf && (${CONFIGURE}) && ${MAKE}
 
-	cd nx-X11 && ${MAKE} World
+	cd nx-X11 && ${MAKE} World USRLIBDIR=$(USRLIBDIR) SHLIBDIR=$(SHLIBDIR)
```

Comment:
Pointing over `USRLIBDIR` to the `nx-X11` path might help with setting the RPATH correctly (well, to some degree, but more on that later...), but will also have side-effects like installing lint libraries there or generally any library that is installed using `InstallLibrary()`.

Are you sure this is well-tested? Do you have the same files in binary packages before and after that PR?